packages/devextreme/playground/jquery.html

```diff
@@ -32,19 +33,71 @@
     <script type="text/javascript" src="../artifacts/js/dx.all.debug.js" charset="utf-8"></script>
```

Comment:
I think it was mistake commit `jquery.html` file

Reply:
Why are you watching PR not ready to review?